src/components/blinds/blinds.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+import "./style.css";
+import blindClosed from "./blinds-closed.svg";
+import blindOpen from "./blinds-open.svg";
+
+const imgage = [blindClosed, blindOpen];
+
+const Blinds = () => {
+  const [state, setState] = useState(true);
```

Comment:
defaultní stav nemá být true ale to co si předáš ze souboru smartHomeData.js skrz Dashboard (viz zadání)